flake.nix

```diff
@@ -131,6 +131,10 @@
         ibis310 = mkDevShell pkgs.ibisDevEnv310;
         ibis311 = mkDevShell pkgs.ibisDevEnv311;
 
+        quarto = (mkDevShell pkgs.ibisQuartoDevEnv310).overrideAttrs (attrs: {
```

Comment:
This special shell environment will go away and be rolled into the main environment once we switch the live site over to quarto.